src/Termi-Main/main.cpp

```diff
@@ -105,12 +107,13 @@
 
 int main()
 {
-    #if defined _WIN32 || defined _WIN64
-        LoadDynamicLibrary("Termi-GUI.dll", "tmain");
-    #elif defined __APPLE__ || defined __MACH__ || defined __linux__ || \
-        defined __FreeBSD__ || defined __OpenBSD__ || defined __NetBSD__
-        LoadDynamicLibrary("libTermi-GUI.so", "tmain");
-    #endif
+    try {
+        auto func = LoadDynamicLibrary<int(*)(int)>(LIB_NAME, "tmain");
+        func(12);
+    } catch (const std::exception& e) {
```

Comment:
Please remove part of code implementing exceptions, because they are not unnecessary due to following:
a) this is function from C library where exceptions do not exist,
b) errors when loading function is handled in `LoadDynmaicLibrary` function.

Rest of code is excellent, thank you for this idea!